api_app/models.py

```diff
@@ -181,3 +188,60 @@ def delete_file(sender, instance: Job, **kwargs):
     if instance.file:
         if instance.file:
             instance.file.delete()
+
+
+class CustomConfig(models.Model):
+    class Type(models.IntegerChoices):
+        ANALYZER = 1, "Analyzer"
+        CONNECTOR = 2, "Connector"
+
+    type = models.PositiveSmallIntegerField(choices=Type.choices)
+    attribute = models.CharField(max_length=128, blank=False)
+    value = models.CharField(max_length=128, blank=False)
+    organization = models.ForeignKey(
+        Organization, on_delete=models.CASCADE, blank=True, null=True
+    )
+    owner = models.ForeignKey(
+        settings.AUTH_USER_MODEL, on_delete=models.CASCADE, blank=False, null=False
+    )
+    name = models.CharField(max_length=128, blank=False)
```

Comment:
`plugin_name` is more understandable

Reply:
Makes sense. However, I've used it in quite a few places now. Should I perform the refactor?

Reply:
yeah please, I mean, it lasts more this exchange of words than the refactor itself :P

Reply:
done